website/docs/docs/quickstarts/dbt-cloud/starburst-galaxy-qs.md

```diff
@@ -0,0 +1,400 @@
+2. Configure the query editor so it queries your S3 bucket. In the upper right corner of the query editor, select your cluster in the first gray box and select your catalog in the second gray box:
+
+    <Lightbox src="/img/quickstarts/dbt-cloud/starburst-galaxy-editor.png" title="Set the cluster and catalog in query editor" />
+
+3. Copy and paste these queries into the query editor. Then **Run** each query individually. 
+
+    Replace `YOUR_S3_BUCKET_NAME` with the name of your S3 bucket. These queries create a schema named `jaffle_shop` and also create the `jaffle_shop_customers`, `jaffle_shop_orders`, and `stripe_payments` tables: 
+
+    ```sql
+    CREATE SCHEMA jaffle_shop WITH (location='s3://YOUR_S3_BUCKET_NAME/dbt-quickstart/');
+
+    CREATE TABLE jaffle_shop.jaffle_shop_customers (
+        id VARCHAR,
+        first_name VARCHAR,
+        last_name VARCHAR
+    )
+
+    WITH (
+        external_location = 's3://YOUR_S3_BUCKET_NAME/dbt-quickstart/jaffle-shop-customers/',
+        format = 'csv',
+        skip_header_line_count=1
+
+    );
+
+    CREATE TABLE jaffle_shop.jaffle_shop_orders (
+
+        id VARCHAR,
+        user_id VARCHAR,
+        order_date VARCHAR,
+        status VARCHAR
+
+    )
+
+    WITH (
+        external_location = 's3://YOUR_S3_BUCKET_NAME/dbt-quickstart/jaffle-shop-orders/',
+        format = 'csv',
+        skip_header_line_count=1
+    );
+
+    CREATE TABLE jaffle_shop.stripe_payments (
+
+        id VARCHAR,
+        order_id VARCHAR,
+        paymentmethod VARCHAR,
+        status VARCHAR,
+        amount VARCHAR,
+        created VARCHAR
+    )
+
+    WITH (
+
+        external_location = 's3://YOUR_S3_BUCKET_NAME/dbt-quickstart/stripe-payments/',
+        format = 'csv',
+        skip_header_line_count=1
+
+    );
+    ```
+4. When the queries are done, you can see the following hierarchy on the query editor's left sidebar:
+
+    <Lightbox src="/img/quickstarts/dbt-cloud/starburst-data-hierarchy.png" title="Hierarchy of data in query editor" />
+
+5. Verify that the tables were created successfully. In the query editor, run the following queries:
+
+    ```sql
+    select * from jaffle_shop.jaffle_shop_customers;
+    select * from jaffle_shop.jaffle_shop_orders;
+    select * from jaffle_shop.stripe_payments;
+    ```
+
+## Connect dbt Cloud to Starburst Galaxy 
+
+1. Log in to [Starburst Galaxy](https://galaxy.starburst.io/login).
+2. If you haven’t already, set your account’s role to dbt_user. Click your email address in the upper right corner, choose **Switch role** and select **dbt_user**.
+3. Click **Clusters** on the left sidebar.
+4. Find your cluster in the **View clusters** table and click **Connection info**. Choose **dbt** from the **Select client** dropdown. Keep the **Connection information** modal open. You will use details from that model in dbt Cloud.
```

Comment:
"use details from that model" --> "use details from that modal"

Reply:
great catch! 🦅 👀